source/_integrations/neurio_energy.markdown

```diff
@@ -11,17 +11,17 @@ ha_platforms:
   - sensor
 ---
 
-Integrate your [Neurio](https://neur.io/) meter information into Home Assistant. To get an API key and secret, login to your [Neurio account](https://my.neur.io/#settings/applications/register) and register an application. Note the Homepage URL and Callback URL are optional.
+Integrate your [Neurio](https://neur.io/) meter information into Home Assistant. To get an API key and secret, login to your [Neurio account](https://my.neur.io/#settings/applications/register) and register an application. Note the Homepage URL and Callback URL are required but can be "fake" URLs.
```

Comment:
Let's remove the requisites out of the introduction and make it part and make it clear part of the configuration instructions.

Reply:
I removed the sentence about the required fields but I'm not sure what you mean for the second part.  Create a new paragraph describing how to setup the Neurio application in their account settings?

Reply:
The general structure of an integration page is:

- Introduction about the integration and/or products
- Optionally a chapter about prerequisites to use it (e.g., set up devices, tokens and such)
- A chapter about configuring the integration.

This structure is missing, hence now the introduction is kinda minimal and mixed with prerequisites.